src/controllers/Controller.js

```diff
@@ -1,60 +1,13 @@
 import SirenClient from '../siren/SirenClient'
 
-export default class Controller {
-    constructor(router, baseUrl = '') {
-        this.router = router
-        this.baseUrl = baseUrl
-    }
-
+export
+default class Controller {
     siren(json) {
         var siren = new SirenClient(json)
         return siren.view
     }
 
-    get(relativeUrl, handler, name = null) {
-        this.route('GET', this.url(relativeUrl), handler)
-    }
-
-    post(relativeUrl, handler, name = null) {
-        this.route('POST', this.url(relativeUrl), handler)
-    }
-
-    delete(relativeUrl, handler, name = null) {
-        this.route('DELETE', this.url(relativeUrl), handler)
-    }
-
-    patch(relativeUrl, handler, name = null) {
-        this.route('PATCH', this.url(relativeUrl), handler)
-    }
-
-    put(relativeUrl, handler, name = null) {
-        this.route('PUT', this.url(relativeUrl), handler)
-    }
-
-    route(method, path, handler, name = null) {
-        var route = {
-            method,
-            path,
-            config: {
-                handler
-            }
-
-        }
-        if (name)
-            route.config.id = name
-
-        this.router.route(route)
-    }
-
-    url(relativeUrl='') {
-        var url = ('/' + this.baseUrl + '/' + relativeUrl).normalize('/')
-        if (url != '/')
-            url = url.trimEnd('/')
-        return url
-    }
-
-    absoluteUrl(relativeUrl='') {
-        var url = this.router.info.uri.trimEnd('/') + ("/" + relativeUrl).normalize('/')
-        return url
+    url(relativeUrl = '') {
+        return this.server.info.uri.trimEnd('/') + ("/" + relativeUrl).normalize('/')
```

Comment:
It is not readily apparent where `this.server` gets set. Please add a comment mentioning it comes from [here](https://github.com/ritterim/definely/pull/62/files#diff-213b1e17a17e68c19c4069af3156210bR39).